packages/datatrak-web/src/AppProviders.tsx

```diff
@@ -21,16 +21,20 @@ const defaultQueryClient = new QueryClient({
       }
     },
   }),
+  queryCache: new QueryCache({
+    // use the errorToast function to display errors by default. If you want to override this, apply an meta.applyCustomErrorHandling to the mutation or an onError to the query
+    onError: (error: any, query) => {
+      if (!query?.meta || !query?.meta?.applyCustomErrorHandling) {
+        errorToast(error.message);
+      }
+    },
+  }),
   defaultOptions: {
     queries: {
       staleTime: 1000 * 60 * 5, // 5 minutes
       refetchOnWindowFocus: false,
-      retry: 0,
+      retry: false,
```

Comment:
ah great find - I assume that this will fix the other places such as login where the request errors show up multiple times

Reply:
I actually think the errors showing up multiple times is a side effect of various queries being present on different screens and possibly causing re-renders when we have redirects. So from what I could tell, for the most part when we handle surveys not being found before it redirects infinitely, then this issue is handled (90% of the time)